### PR TITLE
Strings

### PR DESCRIPTION
**Pull request by TrOlyaDev**

### CODE REVIEW

File: src/strings/Strings.php [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+{
+    /**
+     * Transform text in snake case into camel case string
+     *
+     * @param string $input
+     * @return string
+     */
+    public function snakeCaseToCamelCase(string $input): string
+    {
+        return lcfirst(str_replace(' ','',ucwords(str_replace('_',' ',$input))));
```
Comment (yderevianko-cartera) [inline src/strings/Strings.php]: please add spaces after each ,

File: src/strings/Strings.php [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+    /**
+     * Mirror each word individually without changing word order
+     *
+     * @param string $input multibyte text
+     * @return string
+     */
+    public function mirrorMultibyteString(string $input): string
+    {
+        $words = explode(' ', $input);
+        $mirrorWords = array();
```
Comment (yderevianko-cartera) [inline src/strings/Strings.php]: = [];

File: src/strings/Strings.php [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+     * repeat the noun twice and connect them together with the first and last letter,
+     * combined into one word (WITHOUT a 'The' in front)
+     *
+     * @param string $noun
+     * @return string
+     */
+    public function getBrandName(string $noun): string
+    {
+        if ($noun[0] === $noun[(strlen($noun)-1)]) {
+            $noun = ucfirst($noun . substr($noun,1));
```
Comment (yderevianko-cartera) [inline src/strings/Strings.php]: please add spaces after each ,

File: src/strings/Strings.php [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+     * when a noun STARTS and ENDS with the same letter:
+     * repeat the noun twice and connect them together with the first and last letter,
+     * combined into one word (WITHOUT a 'The' in front)
+     *
+     * @param string $noun
+     * @return string
+     */
+    public function getBrandName(string $noun): string
+    {
+        if ($noun[0] === $noun[(strlen($noun)-1)]) {
```
Comment (yderevianko-cartera) [inline src/strings/Strings.php]: please add spaces before and after -

**Review by yderevianko-cartera — CHANGES_REQUESTED** [review]

File: src/strings/Strings.php (hunk 1 above)
Comment (TrOlyaDev) [reply src/strings/Strings.php]: all fixed

Comment (TrOlyaDev) [reply src/strings/Strings.php]: fixed

**Review by yderevianko-cartera — APPROVED** [review]

**Pull request merged**